### ISSUE TITLE
[Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.04.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#34174]
#### VFS team name
Debt Resolution

#### VFS product name
Medical Copays (MCP)

#### Point of Contact/Reviewers
Noah Gelman (@noahgelman) - Accessibility

---

### Platform Issue
Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### Issue Details
Heading elements are not in a sequentially-descending order

‘What can you do?” Should be an H2. - Sarah Koomson

### Link, screenshot or steps to recreate
https://googlechrome.github.io/lighthouse/viewer/

### VA.gov Experience Standard
[Category Number 02, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

---

### Platform Recommendation
For introductory purposes, "Your Current copay balances" should serve as H1. "We cant access your current copay balances right now" and "what you can do" should serve as H2's 

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by LinnJS** [comment]
@noahgelman I ran light house & aXe on all pages of our Medical copays application and cannot fine heading level violations anywhere in our application.

![lighthouse-details.png](https://images.zenhubusercontent.com/5dcf313b98e0b800015f2353/19f9a72c-804a-42cd-9b91-9db0bdf3ffdd)

**Comment by AngelaFowler82** [comment]
Which test user should I use to validate this?

**Comment by noahgelman** [comment]
@LinnJS I'll get clarification on this issue and respond back today

**Comment by shiragoodman** [comment]
@Saraharaba8 can you please follow up with @craigwheeler on this? Thank you!

**Comment by noahgelman** [comment]
@craigwheeler Was this page made by your team for Debt Resolution specific? Or is this some kind of global page template you're using

**Comment by craigwheeler** [comment]
hey @noahgelman both pages found at `/your-current-balances` and `/your-current-balances/balance-details` are React apps that were created and maintained by the Debt Resolution Team to render medical copays

**Comment by noahgelman** [comment]
@Saraharaba8 Please refer to the above comment, this ticket is for debt resolution

**Comment by Dr-Pongo** [comment]
@shiragoodman @OptionSelect and I are taking over for Craig. It looks like this ticket could be resolved, or is currently waiting for some additional verification?

**Comment by shiragoodman** [comment]
thanks @Dr-Pongo. Unfortunately Noah Gelman is no longer with our team. I tried to follow along in the comments and I'm not sure if Justin Linn's question on 12/29 was ever answered.

Could you please reach out to @joshkimux and/or @AngelaFowler82 ? They are the dedicated accessibility specialists for your team and should be able to help out. If they're not available, please let me know.

**Comment by Dr-Pongo** [comment]
@shiragoodman So I did some digging, and found the initial issue was inside of an 404 alert message. I updated the heading to correctly follow the standard. While investigating the ticket, @AngelaFowler82 pointed out another heading needed to be updated to flow better as well. Both have been changed and merged in the linked PR above.

**Comment by shiragoodman** [comment]
great! Thanks @Dr-Pongo for handling this. For your reference, our team (Platform Governance) will never close these issue tickets. If you believe that the issue identified has been resolved, we ask that you validate with your Product Owner and then you're free to close the ticket. Let me know if you have any questions - thanks!

**Comment by denisecoveyduc** [comment]
@AngelaFowler82 could you take a look at the changes that @Dr-Pongo made? We can discuss on out next standup.
